src/Interpolations.jl

```diff
@@ -4,6 +4,7 @@ export
     interpolate,
     interpolate!,
     extrapolate,
+    scale,
```

Comment:
Or maybe better is a new function `knots` that applies to any `itp`.

Also, now that I think about it, `scaled(knots, ::GriddedInterpolant)` should probably return another `GriddedInterpolant`, one that shares the data array but has new knots.